checkstyle-tester/diff.groovy

```diff
@@ -161,6 +135,20 @@ def hasUnstagedChanges(gitRepo) {
     return hasUnstagedChanges
 }
 
+def runPlainCheckstyleReportsGeneration(cfg) {
+    generateCheckstyleReport(cfg.localGitRepo, cfg.patchBranch, cfg.patchCheckstyleCfg, cfg.listOfProjects, cfg.reportsDir)
+    generateSummaryIndexHtml(cfg.reportsDir)
```

Comment:
>  I think that having summary page 

I don't know how I keep mixing things up. 
When I said summary report, I meant the normal `launch.groovy` report. I'm sorry again.

http://rveach.no-ip.org/checkstyle/regression/reports/20/checkstyle/index.html I meant this report.